### PR TITLE
Fix settings link in notifications email

### PR DESCRIPTION
**Pull request by faebebin**

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-3047 Notifications email settings link invalid url](https://app.clickup.com/t/2192114/QF-3047)

File: docker-app/qfieldcloud/notifs/cron.py [hunk 1]
```diff
@@ -43,12 +44,14 @@ def do(self):
                     logging.warning(f"{user} has notifications, but no email set !")
                     continue
 
+                QFIELDCLOUD_HOST = os.environ.get("QFIELDCLOUD_HOST", None)
```
Comment (why-not-try-calmer) [inline docker-app/qfieldcloud/notifs/cron.py:47]:
```suggestion
                QFIELDCLOUD_HOST = os.environ.get("QFIELDCLOUD_HOST")
```
The form you used is already built-in.

**Review by why-not-try-calmer — CHANGES_REQUESTED** [review]

File: docker-app/qfieldcloud/notifs/cron.py (hunk 1 above)
Comment (faebebin) [reply docker-app/qfieldcloud/notifs/cron.py]: thanks

Comment (suricactus) [reply docker-app/qfieldcloud/notifs/cron.py]: Don't you have access to `settings.QFIELDCLOUD_HOST`? Ideally random reading from the env should be discouraged outside `settings.py`?

**Review by why-not-try-calmer — APPROVED** [review]

**Pull request merged**